### ISSUE TITLE
`flatten` benchmark is timing out in nightly performance testing

### ISSUE THREAD

**Issue by ronawho** [Bears-R-Us/arkouda#930]
Similar to https://github.com/Bears-R-Us/arkouda/issues/917, the `flatten` benchmark is timing out in nightly perf testing. There are similar issues to https://github.com/Bears-R-Us/arkouda/issues/917, but there are some overheads beyond that too. 

The array creation is somewhat slow because casting to strings is slow -- https://github.com/Bears-R-Us/arkouda/issues/675. I'm looking into optimizing the str cast, but am curious if there's an alternate way to create the input. 

The significantly slower part is the `answer` creation with `answer = ak.array(['{}'.format(i) for i in range(N*3)])`. This will create a large array on the client (which is not practical when the problem size and number of server locales grows) and then transfers it via ZMQ over to the server, which even with recent optimizations will ultimately be limited by the TCP bandwidth between the client and server.

**Comment by pierce314159** [comment]
yeah this makes a lot of sense. I think I'm going to change the answer creation to
```python
answer = ak.cast(ak.arange(N*3), 'str')
```
to keep everything server side and take advantage of the str cast speed up. Any alternate way to create the input will hopefully also apply to the answer creation

**Comment by ronawho** [comment]
Sounds good. Most of the benchmarks don't do validation so if this still ends up being slow, I think we could skip the answer-checking. (That said, I like doing the validation, so hopefully it's fast enough)
